packages/api-explorer/__tests__/AuthBox.test.jsx

```diff
@@ -29,26 +30,13 @@ test('should display a single heading for single auth type', () => {
   const securityTypes = {
     "Header Auth":[{"type":"auth","flows":{"implicit":{"authorizationUrl":"http://petstore.swagger.io/oauth/dialog","scopes":{"write:pets":"modify pets in your account","read:pets":"read your pets"}}},"_key":"petstore_auth"}],  
   }
-  const authBox = mountWithIntl(<AuthBox {...props} securityTypes={securityTypes} />);
+  const security = [{"Header Auth": []}]
+  const authBox = mountWithIntl(<AuthBox {...props} securityTypes={securityTypes} security={security} />);
   const popoverContent = shallowWithIntl(<div>{authBox.find('Popover').prop('content')}</div>)
 
-  expect(popoverContent.find('Tabs').length).toBe(1);
-  expect(popoverContent.find('TabPane').prop('tab')).toBe('Header Auth');
+  expect(popoverContent.find(AuthForm)).toHaveLength(1);
 });
 
-test('should display a heading for each auth type with dropdown', () => {
-  // This object is retrieved from OAS library while running in prod.
-  const securityTypes = {
-    "OAuth2 Auth":[{"type":"oauth2","flows":{"implicit":{"authorizationUrl":"http://petstore.swagger.io/oauth/dialog","scopes":{"write:pets":"modify pets in your account","read:pets":"read your pets"}}},"_key":"petstore_auth"}],
-    "Header Auth":[{"type":"auth","flows":{"implicit":{"authorizationUrl":"http://petstore.swagger.io/oauth/dialog","scopes":{"write:pets":"modify pets in your account","read:pets":"read your pets"}}},"_key":"petstore_auth"}],  
-  }
-
-  const authBox = mountWithIntl(<AuthBox {...props} securityTypes={securityTypes} />);
-  const popoverContent = shallowWithIntl(<div>{authBox.find('Popover').prop('content')}</div>)
-
-  expect(popoverContent.find('TabPane').length).toBe(2);
-  expect(popoverContent.find('TabPane').map(pane => pane.prop('tab'))).toEqual(['OAuth2 Auth', 'Header Auth']);
-});
 
 test.skip('should display a dropdown for when multiple oauths are present', () => {
```

Comment:
It is no more in this component, right?

Reply:
I honestly do not know, this test has been skipped 3 years ago :sweat_smile: 

There are several skipped tests in the whole project

Reply:
We could start and remove them but I don't feel very confident right now

Reply:
The `dropdown with multiple oauths` is moved to another component, so we could remove. Skipped test are quite unuseful, so I think it's not a problem

Reply:
yep I agree, my problem is that there's plenty of skipped tests (that have been skipped in the original project) and we can't say whether skipped tests have been skipped for bugs that we might encounter or just things that have been left incomplete.. Anyhow I'm removing the ones in this component since we've thoroughly changed it